protocol/cosi.go

```diff
@@ -74,6 +75,9 @@ type CommitmentHook func(in []abstract.Point) error
 // challenge is received
 type ChallengeHook func(ch abstract.Scalar) error
 
+// ResponseHook allows registering a handler when the response is calculated
```

Comment:
Yep the commitment hook receives the children's commitment, so it's the same. But the comment say here `when the response is calculated`  =>  the response is calculated after.